CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -898,7 +937,8 @@ public let DECL_NODES: [Node] = [
       Child(
         name: "attributes",
         kind: .collection(kind: .attributeList, collectionElementName: "Attribute", defaultsToEmpty: true),
-        nameForDiagnostics: "attributes"
+        nameForDiagnostics: "attributes",
+        documentation: "Attributes that are attached to the func declaration."
```

Comment:
Let’s spell out *function* here because we’re using it as a normal english term instead of as a Swift keyword.

```suggestion
        documentation: "Attributes that are attached to the function declaration."
```